pagopa/pn-frontend-e2e
# Pn-9258-notifica-analogica-stato-consegnata

## Short description

PA sends an analog notification that changes to the delivered state

## List of changes proposed in this pull request

- implemented new test
- implemented new step to check the sending to the physical domicile

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@invioNotificaAnalogicaStatoConsegnata"` and verify the test works fine.
